.github/workflows/acceptance-tests.yml

```diff
@@ -189,6 +189,44 @@ jobs:
         if: failure() || cancelled()
         id: failure_debian
         run: echo '::set-output name=FAILURE_TAG_DEBIAN::true'
+  test-npm-releases:
+    name: NPM acceptance testing
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    outputs:
+      FAILURE_TAG_NPM: ${{ steps.failure_npm.outputs.FAILURE_TAG_NPM }}
```

Comment:
[Nit] Could we maintain consistency for these output tags in all these tests, so we can grep easier later on? Like renaming `FAILURE_TAG_NPM` to `ACCEPTANCE_FAILURE_TAG` throughout npm, scoop, brew, etc? 

Since the step name already signifies the kind of failure, I feel these could be redundant. Also we could grep at a later point using something like:

```
 ${{ contains(steps.*.ACCEPTANCE_FAILURE_TAG, 'true') }}
  // Need to test the wildcard match here
 ```
 
  [Contains Expression](https://docs.github.com/en/actions/learn-github-actions/expressions#contains)

Reply:
It's good suggestion let me change that.